docker-compose.yml

```diff
@@ -132,11 +117,6 @@ services:
       - --v=1
     labels:
       kompose.service.type: LoadBalancer
-    healthcheck:
-      test: ["CMD", "curl", "-f", "http://localhost:8081/readyz"]
-      interval: 30s
-      timeout: 10s
-      retries: 5
```

Comment:
These values do not match those used by Docker's HEALTHCHECK directive. Is that ok?

Reply:
I'm ok reverting to the defaults for the health check.  These values have currently produced checks that fail a bit too quickly